bigquery2/bigquery_endpoint.bal

```diff
@@ -487,10 +488,6 @@ public remote function Client.getAccessTokenFromServiceAccount(string keyStoreLo
 
 public function Client.init(BigqueryConfiguration bigqueryConfig) {
     http:AuthConfig? authConfig = bigqueryConfig.clientConfig.auth;
-    if (authConfig is http:AuthConfig) {
-        authConfig.refreshUrl = REFRESH_URL;
-        authConfig.scheme = http:OAUTH2;
-    }
```

Comment:
Why are we removing this and since we are removing this do we need L490 now?

Reply:
since the user will provide the scheme in the authConfig, I have removed.. 
But I'll revert this change (authConfig.scheme = http:OAUTH2;)

But for refreshUrl, I have tried like this
authConfig.config.refreshUrl = REFRESH_URL;

but error like this
error:wso2/bigquery2:0.8.2::bigquery_endpoint.bal:492:9: invalid operation: type 'ballerina/http:BasicAuthConfig|ballerina/http:OAuth2AuthConfig|ballerina/http:JwtAuthConfig' does not support field access

Reply:
Fixed